### DISCUSSION TITLE
How to send smtp mail to mock after update 3.2.1Final

### DISCUSSION THREAD

**Discussion by LaravZ** [quarkusio/quarkus#35357] [Q&A]
Hi,

I am using Camel Quarkus for a project to send mails using the camel component: ```.to("smtp://{host}");```, instead of the Quarkus Mailer.
For local testing I would use 'smtp.example.com:465' as a host. This would allow the mails to be caught by the org.jvnet.mock-javamail Mailbox. This has always worked until update 3.2.1Final. I've looked into the changes of this update, but could not find a clear indication of anything major having changed on this front. The error is a simple: 
```MailConnectException: Couldn't connect to host, port: smtp.example.com, 465; timeout 1000; Caused by: java.net.UnknownHostException: smtp.example.com```

Any ideas on what has changed to break this setup? And on how to fix this without Quarkus Mailer?

[a comment by an automated account is not shown]

**Comment by maxandersen** [comment]
What fails today? Any exception/logs?

**Reply by LaravZ** [reply]
We're getting ```MailConnectException: Couldn't connect to host, port: smtp.example.com, 465; timeout 1000; Caused by: java.net.UnknownHostException: smtp.example.com``` exceptions.
This makes sense in a way since we don't specify that a mock should be used. However as stated, this has always worked prior to the update. There have been no other changes or updates that could influence this issue.

**Comment by gsmet** [comment]
I think it's probably a question you should ask to the Camel Quarkus team.

**Reply by LaravZ** [reply]
Yes, but since it was a Quarkus update (not a Camel Quarkus update) that broke this I was wondering whether I could be pointed towards the change that introduced this.
That would make it easier for me to drop the question at Camel Quarkus.

**Reply by gsmet** [reply]
It's hard to know as I don't know what Camel Quarkus is doing under the hood. Is it working with 3.2.0.Final? 3.2.1.Final is the first version that broke it?

**Reply by LaravZ** [reply]
3.2.1.Final is the first version that broke it. I think Camel Quarkus is not pinned to a specific version beyond 3.x.x, but will look into it.

**Reply by gsmet** [reply]
If it can help, Camel Quarkus was upgraded from `3.0.0-M2` to `3.0.0-RC1` in Quarkus `3.2.1.Final`

**Reply by LaravZ** [reply]
Version (2.16.x) of Camel Quarkus depends on Camel at version 3.20.1 Quarkus at version 2.16.0.Final. This is not enforced though.

There is a version 3.x.x in the works that will depend on Camel at version 4.x.x. and Quarkus at version 3.2.x.
